<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="ChrisLabel" name="chrisLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>The quick brown fox likes CSE 335</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="LineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="CBBox" name="comboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Family</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="RadioButton" name="radioButton_2">
      <property name="text">
       <string>Arial</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="RadioButton" name="radioButton_3">
      <property name="text">
       <string>Times New Roman</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="RadioButton" name="radioButton">
      <property name="text">
       <string>Comic Sans MS</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Font Size</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="HorizontalSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ChrisLabel</class>
   <extends>QLabel</extends>
   <header location="global">chrislabel.h</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSlider</class>
   <extends>QSlider</extends>
   <header location="global">horizontalslider.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CBBox</class>
   <extends>QComboBox</extends>
   <header location="global">cbbox.h</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header location="global">radiobutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
